src/Uno.UI/Controls/BindableUIView.iOS.cs

```diff
@@ -43,16 +43,34 @@ public override void SubviewAdded(UIView uiview)
 
 		internal List<UIView>.Enumerator GetChildrenEnumerator() => _shadowChildren.Materialized.GetEnumerator();
 
-		public override void WillRemoveSubview(UIView uiview)
+		public override void AddSubview(UIView view)
 		{
-			base.WillRemoveSubview(uiview);
+			// As iOS will invoke the MovedToWindow on the 'view' (which will invoke the Loaded event)
```

Comment:
Is this supposed to be there ?

Reply:
not at all,  wrong commit (cherry-pick) in the wrong branch.